PocketKit/Sources/PocketKit/PocketSceneDelegate.swift

```diff
@@ -22,7 +22,8 @@ public class PocketSceneDelegate: UIResponder, UIWindowSceneDelegate {
                         ),
                         archivedItemsList: ArchivedItemsListViewModel(
                             source: Services.shared.source,
-                            tracker: Services.shared.tracker.childTracker(hosting: .myList.archive)
+                            tracker: Services.shared.tracker.childTracker(hosting: .myList.archive),
+                            archiveService: Services.shared.source.makeArchiveService()
```

Comment:
Grab this archive service from `Source` during initialization instead of requiring it to be passed in